package.json

```diff
@@ -1,16 +0,0 @@
-{
```

Comment:
How to use .gitignore and not sync files such as node_modules.

Also, What are other things we use .gitignore for.